ats/management.py

```diff
@@ -307,8 +309,9 @@ def finalReport(self):
         if not configuration.options.skip:
             log("""
 ATS SUMMARY %s""" % datestamp(long_format=True), echo=True)
-            self.summary(log)
+            clean_run = self.summary(log)
```

Comment:
`clean_run` could be confused with you needed to "clean" the run.  `successful_run` is less ambiguous.

Reply:
I agree, the variable name has been changed.